packages/rule-utils/src/colors.ts

```diff
@@ -58,9 +55,11 @@ export function colorToString(color: CSSColorValue | string, alphaOverride?: str
   alpha = alphaOverride ?? alpha
   type = type.toLowerCase()
 
-  // Comma separated functions
-  if (['hsla', 'hsl', 'rgba', 'rgb'].includes(type))
-    return `${type.replace('a', '')}a(${components.join(',')}${alpha == null ? '' : `,${alpha}`})`
+  if (['hsla', 'rgba'].includes(type))
+    return `${type}(${components.join(', ')}${alpha == null ? '' : `, ${alpha}`})`
```

Comment:
No need space after comma, just as the previous logic.

Reply:
> No need space after comma, just as the previous logic.逗号后不需要空格，就像前面的逻辑一样。

Wouldn't it be more organized if there are spaces? I refer to tailwind's processing of color.

Reply:
I personally think this doesn't matter since the browser doesn't care about it at all. If less space means a smaller file size, then I agree with that, if not, I'd stand with adding the space to make it look better.